frontend/src/components/PrivateComponent/privatecomponenet.tsx

```diff
@@ -0,0 +1,8 @@
+import { Navigate,Outlet } from "react-router-dom";
+
+const PrivateRoute=()=>{
+    const auth=localStorage.getItem('auth');
```

Comment:
how are u checking the expiration of token, please consider implementing refresh-token in future